beyond_django/urls.py

```diff
@@ -14,10 +14,20 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 from msgboard import views
 
+
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', views.board, name='board')
+    path('', views.landing, name='landing'),
```

Comment:
I think the main page should still be the board page. And:
1. We should only show the for for adding posts to logged-in users.
2. When accessed publicly the main page should show a "log in to post" link that leads to the login/signup page.

Reply:
Ack